src/RA_Dlg_Memory.cpp

```diff
@@ -341,9 +341,9 @@ bool MemoryViewerControl::OnEditInput(UINT c)
 
         if (g_MemBookmarkDialog.GetHWND() != nullptr)
         {
-            const MemBookmark* Bookmark = g_MemBookmarkDialog.FindBookmark(nByteAddress);
-            if (Bookmark != nullptr)
-                g_MemBookmarkDialog.WriteFrozenValue(*Bookmark);
+            auto Bookmark = g_MemBookmarkDialog.FindBookmark(nByteAddress);
+            if (Bookmark != std::end(g_MemBookmarkDialog))
+                g_MemBookmarkDialog.WriteFrozenValue(ra::at(nByteAddress, Bookmark->second));
         }
```

Comment:
This was honestly all you had to do to use iterators and not get an exception thrown.